File: app/src/main/java/com/ktcool/test/SecondActivity.java
package com.ktcool.test;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.ktcool.R;
import com.ktcool.annotation.Router;

@Router(path = "/app/secondActivity")
public class SecondActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_search);
    }
}

File: app/src/main/java/com/ktcool/test/SecondActivity.kt
package com.ktcool.test

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class SecondActivity : AppCompatActivity() {
    private val C.length: Int
        get() {
            return name.length
        }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val c = C()
        c.method1()
        c.method2()
        MyPrint.print(c.name + " : " + c.length)

        val d = D()
        d.method1()
        d.method2()
    }
}